src/index.js

```diff
@@ -37,23 +37,23 @@ export default class ReactPasswordStrength extends React.Component {
     const { changeCallback, minScore, userInputs } = this.props;
     const password = this.refs['ReactPasswordStrength-input'].value;
 
-    let score;
+    let score = 0;
+    let result = null;
 
     // always sets a zero score when min length requirement is not met
     // which avoids unnecessary zxcvbn computations (they require quite lots of CPU)
-    if (this.isTooShort(password)) {
-      score = 0;
-    } else {
-      score = zxcvbn(password, userInputs).score;
+    if (!this.isTooShort(password)) {
+      result = zxcvbn(password, userInputs);
+      score = result.score;
     }
 
     this.setState({
       isValid: score >= minScore,
       password,
-      score,
+      score
     }, function() {
       if (changeCallback !== null) {
-        changeCallback(this.state);
+        changeCallback(this.state, result);
```

Comment:
please update the `README` to document that we are passing back the computed `result` and a short blurb about what it is

Reply:
Can do, and reverting those small changes!